### PR TITLE
Improved table of contents configuration

### PR DESCRIPTION
**Pull request by elpoelma**
This PR introduces some changes in how you can pass a custom configuration to the plugin.

You can now pass a custom configuration to both the plugin and inline component as explained in the updated README.

### CODE REVIEW

File: addon/components/inline-components/table-of-contents.js [hunk 1]
```diff
@@ -43,18 +43,15 @@ export default class TableOfContentsComponent extends Component {
   }
 
   extractOutline(node) {
+    console.log(this.config);
```
Comment (lagartoverde) [inline addon/components/inline-components/table-of-contents.js]: forgotten console.log

**Review by lagartoverde — COMMENTED** [review]
Not really sure i understand this PR, why are we reverting to the old configuration?

File: addon/components/inline-components/table-of-contents.js (hunk 1 above)
Comment (elpoelma) [reply addon/components/inline-components/table-of-contents.js]: > Not really sure i understand this PR, why are we reverting to the old configuration?

Yes to the ones without RDF namedNodes, as inline-components cannot serialize them fully.

Comment (lagartoverde) [reply addon/components/inline-components/table-of-contents.js]: but wasn't the `SAY('')` thing what we used to resolve the bug with the unexpanded uris?

**Pull request merged**